sklearn/cluster/_agglomerative.py

```diff
@@ -151,29 +151,29 @@ def ward_tree(X, *, connectivity=None, n_clusters=None, return_distance=False):
 
     Parameters
     ----------
-    X : array, shape (n_samples, n_features)
+    X : array of shape (n_samples, n_features)
```

Comment:
If `X` can be a `list` use `array-like`. Otherwise, `ndarray`.

Reply:
ok, I'm assuming X can be a list since the first operation is, X = np.asarray(X) - is that correct?

Reply:
That's right :)

Reply:
Great, thanks.